src/main/resources/swagger/api-docs.yaml

```diff
@@ -211,6 +211,43 @@ paths:
           description: Workspace does not exist
         500:
           description: Internal Server Error
+  /workspaces/{workspaceNamespace}/{workspaceName}/entities/copy:
+    post:
+      tags:
+      - Entities
+      operationId: copyEntities
+      summary: |
+        Copy entities from one workspace to another
+      produces:
+        - application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: body
```

Comment:
how about removing it from the body, then? Make the url parameters the destination workspace, and include the source workspace in the body? You'd have to have extra manipulation code inside the endpoint to translate into the rawls format, but it makes for a better end-user API.